app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -50,13 +47,31 @@ export const USER_IMG_URL =
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: null,
+  tags: [],
 };
 
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
-  contactList: yup.string().oneOf(CONTACTS),
-  tagList: yup.string().oneOf(TAGS),
+  contact: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string().oneOf(CONTACT_OPTIONS.map(contact => contact.label)),
+      value: yup.string().oneOf(CONTACT_OPTIONS.map(contact => contact.value)),
+    })
+    .required("Please select a contact"),
+  tags: yup
+    .array()
+    .of(
+      yup.object().shape({
+        label: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.label)),
+        value: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.value)),
+      })
+    )
+    .min(1, "Every note should have atleast 1 tag")
```

Comment:
This validation message is too long and verbose. Keep it compact.

Reply:
> @aagneyanNamboodiri The validation messages should have some consistency in the way they are being pointed out to the end user.
> 
> Can you comment on what you feel about the consistency in the sentence construction of the following messages? Do you see a scope for improvement? <img alt="image" width="468" src="https://user-images.githubusercontent.com/31244999/191109963-1571d01f-de40-4a67-ad49-395cb1a7c75a.png">

The error messages do seem inconsistent. 
I think just saying "Please select a tag" will be sufficient here. Also, it is similar to the previous for select's validation message which makes it easier for the user to understand.